docs/multicluster/upgrade.md

```diff
@@ -37,6 +37,35 @@ for the feature in new version.
 It should have no impact during upgrade to those imported resources like Service, Endpoints
 or AntreaClusterNetworkPolicy.
 
+## Upgrade from a version prior to v1.13
+
+Prior to Antrea v1.13, the `ClusterClaim` CRD is used to define both the Cluster ID and
+ClusterSet ID for a ClusterSet. Since Antrea v1.13, the `ClusterClaim` CRD is removed, and
+the `ClusterSet` CRD solely defines a ClusterSet. The name of a ClusterSet CR must match the
+ClusterSet ID, and a new `clusterID` field specifies the Cluster ID.
+
+To upgrade from a version older than v1.13, once you apply the latest manifests for an
+existing ClusterSet, you need to recreate the `ClusterSet` CR manually with `clusterID`, and
```

Comment:
I did not get what you mean. You said "The old CRs will take no affect." After upgrade, mc-controller will not recognize old version CRs? Then the previous ClusterSet is lost? Then what will happen for MC Gateway, ResourceExport/Import, etc.? How about ongoing MC traffic?

Reply:
Existing traffic shouldn't be impacted if there is no Gateway, Endpoint changes etc. Any new changes for exported Services or Gateways or Pods will be ignored during upgrade. Previous ClusterSet won't be processed since they are different versions unless we add a ClusterSet conversion to update it to a new version's CR. 
If we add a ClusterSet conversion, we need get the ClusterID from a ClusterClaim, I feel it don't make sense to do a version upgrade based on two CRs. So I feel we may alert users that this is a breaking change and need to add new ClusterSet manually, what's your thoughts?

Reply:
After the new MC Controller starts, nothing will happen if it does not see a valid ClusterSet CR?

And what will happen if there is any change in the ClusterSet (in local or remote or leader cluster) before the new ClusterSet CR is created? E.g. Service Endpoints change, a MC Service or Gateway is deleted / created, a new Pod is created that affects label identities, ...

Reply:
Yes, if we didn't provide a ClusterSet conversion webhook, no remote common area will be created in antrea-mc-controller since MC controller won't recognize the old ClusterSet CR. Any change event reconcile will fail until a valid ClusterSet CR is created. The failed events will be re-queued and retry until a valid remote common area is available.

Reply:
Nothing will happen for existing ResourceExport, ResourceImport, ServiceExport, etc.? For example, leader wont remove any Export/Import for a member? If that is the case, does it mean nothing will happen for existing Export/Import either, even we remove a ClusterSet CR from leader or member? Then how can users cleanup a ClusterSet? They need to manually delete all Export/Import CRs in leader and members? That does not sound like a good experience.

For upgrade, I wonder why not create a new version ClusterSet CR, before updating MC Controller deployment?

Reply:
@jianjuns yeah, I am afraid user needs to clean existing resources by themself at the moment, I think we discussed resources clean up before when a member cluster leave a ClusterSet, but there was no conclusion and was postponed. I feel we may prioritize this part in next release.
For upgrade, user can create a new version of CR before updating deployment, but I feel we may need to provide a way to split CRDs update and deployment update first. For now, when user is trying to run `kubectl apply` with new manifests, both CRDs and MC controller deployment will be updated almost at the same time.
I will check if there is a proper way to split them, let me know if you have other ideas. Thanks.

Reply:
After run some verification locally, I think we need keep both versions for ClusterSet if we want to support upgrade for existing ClusterSet since the CR of version v1alphav1 is still kept in the cluster.
I feel we may still need a ClusterSet conversion controller based on existing ClusterClaims and ClusterSet. Maybe not a webhook, just a controller like stale_controller to watch two CRs and create a new version of ClusterSet if possible.
what's your suggestion?

Reply:
For cleanup, it should be ok for users for remove CRs created by them, but not good to require them to remove CRs created by MC Controller. Let us track that change. And before we have an automated way, could we at least document steps to remove all CRs?

For upgrade, sure probably let us add a one time step to create new version CRs. Let us do it in another PR.

Reply:
Doc updated, add a few commands for user as a reference to create a new version CR in the doc.